crds/vshn.appcat.vshn.io_vshnredis.yaml

```diff
@@ -78,6 +78,16 @@ spec:
                           pattern: ^([0-1]?[0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9])$
                           type: string
                       type: object
+                    restore:
+                      description: Rewstore contains settings to control the restore of an instance.
```

Comment:
Looks like a typo. But that should be auto generated and I don't see the typo in the comment. Please double check and regenerate the CRD.